General.py

```diff
@@ -0,0 +1,41 @@
+# Q1
+def sum_of_array(array):
```

Comment:
This question was for summing together the even numbers which is not included in your implementation